### PR TITLE
Tweak MangaInfoHeader

### PR DESCRIPTION
**Pull request by ghostbear**
- Make margin between the transparent toolbar and cover more match 1.x
- Fixes from when the view was redone with MotionLayout

| Old | New |
|---|---|
| Screenshot_1629584325 | Screenshot_1629585629 |

### CODE REVIEW

File: app/src/main/res/layout/manga_info_header.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
         android:layout_width="100dp"
         android:layout_height="0dp"
         android:layout_marginStart="16dp"
-        android:layout_marginTop="8dp"
+        android:layout_marginTop="24dp"
```
Comment (arkon) [inline app/src/main/res/layout/manga_info_header.xml:39]: I actually feel like this is too much now that it's edge-to-edge and there's overall more content getting shown. I tried 16dp locally and it looked pretty good imo.

Comment (ghostbear) [reply app/src/main/res/layout/manga_info_header.xml:39]: 16dp is what 1.x uses so I'm okay with that

**Comment by vengiare** [comment]
The buttons below look odd now. Is bottom margin really needed?

**Comment by vengiare** [comment]
Now it looks cramped compared to current one tho
image

**Pull request merged**